org.jacoco.core.test/src/org/jacoco/core/tools/ExecDumpClientTest.java

```diff
@@ -126,6 +126,12 @@ public void testReset() throws IOException {
 		assertTrue(resetRequested);
 	}
 
+	@Test(expected = IOException.class)
+	public void testNopServer() throws IOException {
```

Comment:
@Godin Good catch! Needed to insert a Thread.sleep() to reproduce this, but I was able to reproduce it. Fixed in 42b4f9f57d4403778f49c89a5790f287a06f1572 ✅ 

I didn't change the method name originally, as the code change was a 1:1 patch from #525. Now as we needed to adjust the code I also change the method name as suggested. To avoid merge conflicts I will revert the corresponding change in #525, correct?